source/retryable-writes/tests/insertMany.yml

```diff
@@ -47,28 +47,3 @@ tests:
                     - { _id: 1, x: 11 }
                     - { _id: 2, x: 22 }
                     - { _id: 3, x: 33 }
-    -
-        description: "InsertMany fails after multiple network errors"
```

Comment:
I removed this test because it sets the fail point to "alwaysOn", so it loops forever after the changes in DRIVERS-555. Also, we're already testing insertMany with multiple network errors in `insertMany-serverErrors.yml` (test name is `InsertMany succeeds after two connection failures`), so I don't think removing this loses any test coverage.